lib/prawn.rb

```diff
@@ -37,7 +37,7 @@ module Prawn
   # @return [Boolean]
   attr_accessor :debug
 
-  module_function :debug, :debug=
+  module_function :debug, :debug= # rubocop:todo Style/AccessModifierDeclarations
```

Comment:
I'd suggest using `rubocop: disable` as this (and the other one, too) is a part of public API that is not going to change.